.travis.yml

```diff
@@ -24,11 +24,9 @@ script:
     # - The source code from travis is 'mounted' as /home/nuto/source via the -v option in the docker run command above
     - docker exec dock cmake -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_OPENMP=$OPENMP -DCMAKE_CXX_FLAGS=$COVERAGE ../source
     - docker exec dock make -j2 unit
+    - if [[ "$COVERAGE" == "--coverage" ]]; then docker exec dock bash -c "cd ../source && scripts/upload_coverage.sh unittests"; fi
```

Comment:
Just curious: Is the order of commands important here? So could we, in theory, collect all the coverage stuff at the end? Would there be drawbacks?

Reply:
Yes, the order matters. After the unit tests are run, the coverage is uploaded and the `.gcada` files are deleted. Then, the next set of tests is run, and the coverage uploaded again. So unfortunately, I don't see a way to move it to the end.